src/components/Comments/ListComment.js

```diff
@@ -15,14 +15,16 @@ const ListComment = ({
   findCommentAndUpdate,
   hideLeftConnector,
   navigation,
-  updateVote
+  updateVote,
+  feedId
 }) => {
   return (
     <TouchableWithoutFeedback key={index} onLongPress={() => onCommentLongPressed(item, 0)}>
       <View>
         <View key={`p${index}`}>
           {item.user ? (
             <Comment
+              feedId={feedId}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new prop `feedId` is being passed to the `Comment` component. Ensure that this prop is properly validated and used within the `Comment` component. If it's not optional, make sure it's always provided when the `Comment` component is used.

```diff
<Comment
+  feedId={feedId}
   indexFeed={indexFeed}
   key={`p${index}`}
   comment={item}
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->